src/components/Footer/Footer.module.css

```diff
@@ -27,32 +27,22 @@
 .author {
   color: white;
   font-weight: 500;
-  font-size: 16px;
-  line-height: 28px;
   margin-bottom: 30px;
 }
 
 .wrapper {
   display: flex;
-  gap: 196px;
+  gap: 150px;
   justify-content: space-between;
   align-items: center;
 }
 
 .quote {
-  height: 200px;
+  height: 20%;
```

Comment:
Usually we should avoid using % as measure units, @ssukovic maybe we should consider defining range of size variables i.e. var(--space-100) and unify defined measures with px units.